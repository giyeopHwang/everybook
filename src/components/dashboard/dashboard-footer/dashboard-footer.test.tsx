import { renderWithWrapper } from '@/test/test-utils';
import DashboardFooter from './dashboard-footer.components';

describe('DashboardFooter', () => {
  it('renders DashboardFooter component', () => {
    const dashboardFooter = renderWithWrapper(<DashboardFooter />);
    expect(dashboardFooter).toMatchSnapshot();
  });
});
