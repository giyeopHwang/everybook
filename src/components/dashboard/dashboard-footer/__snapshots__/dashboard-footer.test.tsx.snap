// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DashboardFooter > renders DashboardFooter component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="sc-eDvShL ellwLC"
      >
        <div
          class="sc-dkrGBB hWbfRc"
        >
          <span>
            이달의예산
          </span>
          <div
            class="sc-bcXGCL jXLNxY"
          >
            <div
              class="sc-gswONk dIRKrZ"
            />
          </div>
          <span
            class="sc-hLBatu eQTfWq"
          >
            3,000,000
          </span>
          <span>
            /
          </span>
          <span>
            5,000,000
          </span>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="sc-eDvShL ellwLC"
    >
      <div
        class="sc-dkrGBB hWbfRc"
      >
        <span>
          이달의예산
        </span>
        <div
          class="sc-bcXGCL jXLNxY"
        >
          <div
            class="sc-gswONk dIRKrZ"
          />
        </div>
        <span
          class="sc-hLBatu eQTfWq"
        >
          3,000,000
        </span>
        <span>
          /
        </span>
        <span>
          5,000,000
        </span>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
